ui/main/src/app/services/users/UsersService.ts

```
@@ -0,0 +1,242 @@
+                if (userResponse.status === ServerResponseStatus.OK) {
+                    return userResponse.data;
+                } else {
+                    ErrorService.handleServerResponseError(userResponse);
+                    return [];
+                }
+            })
+        );
+    }
+
+    public static getAll(): Observable<User[]> {
+        return this.queryAllUsers();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)